docs/development/modernizing-existing-add-ons.md

```diff
@@ -0,0 +1,68 @@
+<!--
+    This source file is part of the open source project
+    ExpressionEngine User Guide (https://github.com/ExpressionEngine/ExpressionEngine-User-Guide)
+
+    @link      https://expressionengine.com/
+    @copyright Copyright (c) 2003-2022, Packet Tide, LLC (https://packettide.com)
+    @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+-->
+
+# Modernizing Existing Add-ons
+
+
+
+The ExpressionEngine 7.2 release brough a new add-on developemnt approach, which generates the majority of the add-ons needed files and struture.  It also bring additional organization to how add-ons are developed.
+
+NOTE: These changes do NOT break existing add-ons.  The old development methodologies will still work.
+
+The idea of a plugin, module, extension, fieldtype, etc. no longer exist.  Instead an add-on can have a template tag, or a field-type, or an extention.
```

Comment:
@TomJaeger It appears as though the idea still exists, but has changed. May be better language around that.

Extension is misspelled.

Reply:
@dougblackjr can you review what I changed it to? Does that sum it up well, or do you think it would be better worded in a different way?